client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/AutoAllocate.tsx

```diff
@@ -39,7 +39,10 @@ export const AutoAllocate = () => {
   // pack size which stops you entering the required quantity.
   // See https://github.com/msupply-foundation/open-msupply/issues/2727
   const debouncedAllocate = useDebounceCallback(
-    quantity => autoAllocate(quantity, format, t),
+    quantity => {
+      const allocated = autoAllocate(quantity, format, t);
+      setIssueQuantity(allocated);
```

Comment:
Have to call setIssueQuantity again here with returned value, same reason as the NumberInputCell, using buffer state